uno.platform.uno-calculator.appdata.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>uno.platform.uno-calculator</id>
+  <launchable type="desktop-id">uno.platform.uno-calculator.desktop</launchable>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Apache-2.0</project_license>
+  
+  <name>Uno Calculator</name>
+  <summary>Linux port of the Microsoft Windows calculator app</summary>
+
+  <developer id="uno.platform">
+    <name>Uno Platform</name>
+  </developer>
+
+  <releases>
+    <release date="2023-09-12" version="1.2.9" />
+  </releases>
+
+  <categories>
+    <category>Utility</category>
+  </categories>
+
+  <description>
+    <p>
+      The open-source, Windows Calculator receives a cross-platform makeover from Uno Platform and now runs on Linux in addition to already running on Android, iOS, macOS and Web (WebAssembly)
+      This is a port of the offical Snap version.
+    </p>
+  </description>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://uno-website-assets.s3.amazonaws.com/wp-content/uploads/2022/03/23133038/UnoCal_Linux.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://dashboard.snapcraft.io/site_media/appmedia/2020/09/Linux-calc-01.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://dashboard.snapcraft.io/site_media/appmedia/2020/09/Linux-calc-02.png</image>
+    </screenshot>
+  </screenshots>
+
+  <url type="homepage">https://platform.uno/uno-calculator/</url>
+  <url type="bugtracker">https://github.com/unoplatform/calculator/issues</url>
+
```

Comment:
Could you add cvs-browser URL. Thanks.

`<url type="vcs-browser">https://github.com/unoplatform/calculator/</url>`

Reply:
Done